rke/structure_rke_cluster.go

```diff
@@ -505,3 +511,13 @@ func expandRKEClusterFlag(in *schema.ResourceData, clusterFilePath string) clust
 
 	return obj
 }
+
+func k8sVersionRequiresCri(kubernetesVersion string) bool {
+
+	version, err := getClusterVersion(kubernetesVersion)
+	if err != nil {
+		// Ignoring the error, if the k8sVersion is not valid it should fail during the validation of the field.
+		log.Debug("invalid kubernetes version")
+	}
+	return semver.MustParseRange(">= 1.24.0-rancher0")(version)
```

Comment:
Can we define `semver.MustParseRange(">= 1.24.0-rancher0")` as a const at the module level?

Reply:
Did it as a var, can't set it a const. 

Just to understand the logic / reason, why are we using this as a var/const ?